pom.xml

```diff
@@ -377,6 +382,10 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>versions-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-invoker-plugin</artifactId>
+      </plugin>
```

Comment:
adding invoker plugin as a report generator for the maven site. It will publish a report with the integration-test projects ran, their status (success or failed) and their build time